flow-server/src/main/resources/vite.generated.ts

```diff
@@ -18,12 +18,13 @@ import brotli from 'rollup-plugin-brotli';
 import replace from '@rollup/plugin-replace';
 import checker from 'vite-plugin-checker';
 
-const appShellUrl = '.';
+const appShellUrl = '.'
```

Comment:
I suggest keeping the semicolon :)
```suggestion
const appShellUrl = '.';
```

Reply:
done